src/constants/index.js

```diff
@@ -1,6 +1,7 @@
 module.exports = {
   AFRAME_INJECTED: 'aframe-injected',
   DEFAULT_CAMERA_HEIGHT: 1.6,
+  DEFAULT_HAND: 'right',
```

Comment:
but the `hand` property is something `gamepad` specific. This is a global variable at the library level. `DEFAULT_HAND` does not give you much info when looking at the list of global variables with no context.